test/xkeyboard-config-test.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python3
+
+import sys
+import subprocess
+import xmltodict
```

Comment:
It would be better to not need any external dependencies. Using https://docs.python.org/3/library/xml.etree.elementtree.html directly *shouldn't* be too much trouble hopefully.